token-lending/program/src/instruction.rs

```diff
@@ -362,6 +362,22 @@ pub enum LendingInstruction {
         /// liquidity_amount is the amount of collateral tokens to withdraw
         collateral_amount: u64,
     },
+
+    // 16
+    /// Updates a reserve config parameter
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   1. `[writable]` Reserve account - refreshed
+    ///   2 `[]` Lending market account.
+    ///   3 `[]` Derived lending market authority.
+    ///   4 `[signer]` Lending market owner.
+    UpdateReserveConfig {
+        /// Specifies which config to change
+        config_enum: u8,
+        /// New value for the config
+        new_value: u8,
```

Comment:
Hm value type isn't always `u8` (e.g. fees is `ReserveFees`). Maybe we could pass in the entire `ReserveConfig`?

Reply:
Yeah, was talking to devin about that yesterday. Yeah I suppose we can pass in the entire ReserveConfig

Reply:
Do you know of a less brittle way of doing this than just having a agreed upon ordered list of values?
Like, would we ever pass in a serialized json map or something?

Reply:
serialized map isn't that different from agreed upon order